src/hamiltonian.F90

```diff
@@ -505,18 +505,19 @@ subroutine hamiltonian_wigner_seitz(count_pts)
 
     logical, intent(in) :: count_pts
     !! Only count points and return
-
+    integer, parameter :: search_size = 2
```

Comment:
As we discussed, make search_size an input parameter, and make a quick benchmark to see, with a dense k-mesh, if 2 is much slower (then leave 1 as default) or 2 is ok.